libobs/obs-hotkey.c

```diff
@@ -1239,6 +1239,9 @@ static inline void handle_binding(obs_hotkey_binding_t *binding,
 				  uint32_t modifiers, bool no_press,
 				  bool strict_modifiers, bool *pressed)
 {
+	if (pressed)
```

Comment:
`pressed` is a pointer to a boolean, not a boolean.  This will make the function do nothing any time the `pressed` pointer is set.  Are you sure you tested this thoroughly?

Reply:
Yes, it was tested well and no problems came up. Would there be a more proper way to check value of a boolean pointer?

Reply:
I think I got it figured out now. I'm now using the same pressed if statement that is further down in the function and this seems to work as well.